os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -1116,10 +1116,38 @@ void vTaskDelay(int ms)
 {
 	_tizenrt_mdelay_os(ms);
 }
-int _tizenrt_printf(const char *format)
+
+int rtw_printf(const char *format,...)
+{
+	va_list ap;
+	int ret;
+#ifdef CONFIG_DEBUG_LWNL80211_VENDOR_DRV_INFO
+	va_start(ap, format);
+#ifdef CONFIG_LOGM
+	ret = logm_internal(LOGM_NORMAL, LOGM_IDX, LOGM_INF, format, ap);
+#else
+	ret = vsyslog(LOG_INFO, format, ap);
+#endif
+	va_end(ap);
+#endif
+	return ret;
```

Comment:
Sometimes, `ret` can be used without any assignment or initialization.
Then it can return trash value.
How about initializing it?

Reply:
I will resolve in next commit.